src/components/Card.jsx

```diff
@@ -64,45 +81,54 @@ const Card = ({
   linkText,
   imgUrl,
   imageDescription,
+  labels,
+  graphql
 }) => {
   const [isDisabled, setIsDisabled] = useState(false)
   return (
-    <StyledCard>
-      <img
-        id="card-image"
-        src={imgUrl ? imgUrl : defaultCardImage}
-        alt={imageDescription ? imageDescription : 'Description'}
-      />
+    <StyledCard imgUrl={imgUrl}>
+      {/* if you pass true for the graphql prop, the imgURL prop must then be gastbyImageData from a graphQL call which will use the <GastbyImage> component. otherwise pass false to graphql prop and pass a regular image src in the <img> */}
+      {graphql ? <GatsbyImage image={getImage(imgUrl)}/>:
```

Comment:
Interesting solution, but I can see this being hard to maintain and also may break other existing places in the codebase. Can we use a different way to determine if Gatsby image is being passed down? 

For example if `imgUrl` (which may need to be renamed to something more generic like "image") is an Object then it's likely gatbsy image data and if it's a string then it's more likely a url.

Reply:
yes, good call to use typeOf (that's what oyu mean right?).